Mathlib/Logic/Function/OfArity.lean

```diff
@@ -6,6 +6,7 @@ Authors: Mario Carneiro
 
 import Mathlib.Init.Data.Nat.Notation
 import Mathlib.Mathport.Rename
+import Mathlib.Data.Fin.Tuple.Basic
```

Comment:
I would move these results to `Mathlib.Data.Fin.Tuple.Basic` so that this file has very little imports.

Reply:
I put them in a new file, `.Basic` is getting rather long.